Source/WebCore/page/scrolling/mac/ScrollingTreeScrollingNodeDelegateMac.mm

```diff
@@ -267,24 +285,24 @@
 
 void ScrollingTreeScrollingNodeDelegateMac::updateScrollbarPainters()
 {
-    if (m_inMomentumPhase && (m_verticalScrollerImp || m_horizontalScrollerImp)) {
+    if (m_inMomentumPhase && (m_scrollerPair->verticalScroller().scrollerImp() || m_scrollerPair->horizontalScroller().scrollerImp())) {
```

Comment:
Maybe add a helper on `ScrollerPairMac` to ask if there's at least one scrollerImp

Reply:
Fixed.